macros/staging/hubspot/get_hubspot_engagement_columns.sql

```diff
@@ -0,0 +1,16 @@
+{% macro get_hubspot_engagement_columns() %}
+
+{% set columns = [
```

Comment:
I see some columns in here (`owner_id`, `active`, `created_at`) that aren't present in the seed or the staging model, should we remove them here?